### DISCUSSION TITLE
keyboard shortcuts for sponsorblock

### DISCUSSION THREAD

**Discussion by realrogue** [FreeTubeApp/FreeTube#5337] [Q&A]
Hi all, in any other browser with sponsor block enabled i can skip segments that are shown but not auto skipped with the enter key.
In FreeTube, enter just goes in and out of fullscreen.
The other thing is it doesn't show SponsorBlock's usual UI eg. "segment skipped" with an option to unskip.
Is there anything like that in FreeTube? I guess i'm missing something but it's not in any of the short cut pages etc and couldn't see anything in settings.

Also add on question: is there a way to automatically expand the comments section instead of clicking on it every time?

**Comment by 4yman-0** [comment]
> is there a way to automatically expand the comments section instead of clicking on it every time?

I did not find any way to do that in the latest version (v0.21.0 Beta)

**Reply by kommunarr** [reply]
[From the official release page for v0.21](https://github.com/FreeTubeApp/FreeTube/discussions/5298):
> Replace Comment Auto Load setting with all-encompassing Auto Load Next Page setting by @PikachuEXE in https://github.com/FreeTubeApp/FreeTube/pull/4565

![Screenshot_20240626_115153](https://github.com/FreeTubeApp/FreeTube/assets/84899178/10311bec-fe6a-472b-b243-b5dd215fb5d1)

**Reply by 4yman-0** [reply]
Thanks!

**Comment by realrogue** [comment]
can anyone answer my original question?

**Reply by efb4f5ff-1298-471a-8973-3d47447115dc** [reply]
> Hi all, in any other browser with sponsor block enabled i can skip segments that are shown but not auto skipped with the enter key.
In FreeTube, enter just goes in and out of fullscreen.

We dont have SponsorBlock related shortcuts implemented in FreeTube.

> The other thing is it doesn't show SponsorBlock's usual UI eg. "segment skipped" with an option to unskip.
Is there anything like that in FreeTube?

This isnt an option yet but i believe that somebody is working on this as we speak.
